crates/viewer/re_view_spatial/tests/opacity_2d.rs

```diff
@@ -0,0 +1,84 @@
+use re_chunk_store::RowId;
+use re_log_types::TimePoint;
+use re_test_context::TestContext;
+use re_test_viewport::TestContextExt as _;
+use re_viewer_context::{ViewClass as _, ViewId};
+use re_viewport_blueprint::ViewBlueprint;
+
+#[test]
+pub fn test_opacity_2d() {
+    let mut test_context = TestContext::new_with_view_class::<re_view_spatial::SpatialView2D>();
+
+    // Tests
+    // * the heuristics (the RGB should become transparent)
+    // * actual opacity
+
+    {
+        use ndarray::{Array, ShapeBuilder as _, s};
+
+        test_context.log_entity("2d_layering/gradient", |builder| {
+            let (width, height) = (100, 50);
+            let mut image = Array::<u8, _>::zeros((height, width, 3).f());
+            image.slice_mut(s![.., .., 0]).fill(0);
+            image.slice_mut(s![.., .., 1]).fill(255);
+            image.slice_mut(s![.., .., 2]).fill(127);
+
+            builder.with_archetype(
+                RowId::new(),
+                TimePoint::default(),
+                &re_sdk_types::archetypes::Image::from_color_model_and_tensor(
+                    re_sdk_types::datatypes::ColorModel::RGB,
+                    image,
+                )
+                .unwrap()
+                .with_draw_order(1.0),
+            )
+        });
+
+        test_context.log_entity("2d_layering/depth", |builder| {
```

Comment:
Does this only work now because `2d_layering/depth < 2d_layering/gradient` lexicographically? 😬